x-pack/solutions/security/plugins/security_solution/server/lib/telemetry/configuration.ts

```diff
@@ -28,11 +29,15 @@ class TelemetryConfigurationDTO {
   private readonly DEFAULT_INDICES_METADATA_CONFIG = {
     indices_threshold: 10000,
     datastreams_threshold: 1000,
+    indices_settings_threshold: 10000,
```

Comment:
It is needed by https://github.com/elastic/kibana/pull/213822, but I'm adding it here to avoid potential merge conflicts.